utils/tang/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
no, you don't have copyright assignment.  Either drop it completely, or put your own name

Reply:
OK